frontend/src/concepts/modelRegistry/apiHooks/useModelRegistryServices.ts

```diff
@@ -72,17 +83,19 @@ export const useModelRegistryServices = (namespace: string): ModelRegistryServic
       .flatMap((rule) => rule.resourceNames || []);
   }, [rulesReviewLoaded, rulesReviewStatus]);
 
-  const callback = React.useCallback<FetchStateCallbackPromise<ServiceKind[]>>(
-    () =>
-      listServicesOrFetchThemByNames(
-        allowList,
-        accessReviewLoaded,
-        rulesReviewLoaded,
-        namespace,
-        serviceNames,
-      ),
-    [allowList, accessReviewLoaded, rulesReviewLoaded, serviceNames, namespace],
-  );
+  const callback = React.useCallback<FetchStateCallbackPromise<ServiceKind[]>>(() => {
+    if (!namespace) {
+      return Promise.reject(new NotReadyError('No registries namespace could be found'));
+    }
```

Comment:
You don't need this since you added the NotReadyError inside `listServicesOrFetchThemByNames`, you can let that function handle this error case